src/materialized/bin/materialized.rs

```diff
@@ -33,6 +33,7 @@ use failure::{bail, format_err, ResultExt};
 use lazy_static::lazy_static;
 use log::trace;
 use once_cell::sync::OnceCell;
+use std::str::FromStr;
```

Comment:
Please move this to the `std` block above.

Reply:
Actually, it's not needed at all if I switch to using `opt_get` as you suggested.